html/common/aup-autosubmit.js

```diff
@@ -0,0 +1,6 @@
+$('label[for="aup"]').closest('div').click(function(e) {
+  e.preventDefault();
+  $('#aup').attr('checked', 'checked');
+  $('#button').click();
```

Comment:
Instead of relying on the submit button, I would submit the form:

```javascript
$(this).closest('form').submit();
```

Reply:
I had exactly that before but it didn't work when I tested it with a customer and his browser although it worked in mine.

This though has worked consistently so that's why I put it there although I agree it makes more sense to use the line you suggested

Reply:
I don't like the fact that the Continue/submit button is disabled. I would expect a problem when programatically clicking this button more than submitting the form. If you feel confortable with your code, we can keep it like this.

Reply:
I'd feel more comfortable to leave it as-is since the cleaner way has caused me trouble in the deployment where I tested this PR